src/crypto/tls/handshake_test.go

```diff
@@ -330,6 +330,25 @@ func allCipherSuites() []uint16 {
 
 var testConfig *Config
 
+// handshakeTestResult captures the test result of a given handshake.
```

Comment:
This isn't used anywhere, AFAICT. Delete it?

Reply:
Yep, missed that.